<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TCP Server</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>myfile/images/Mabe/celluar.jpg</normaloff>myfile/images/Mabe/celluar.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QSpinBox" name="beat_timer">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="editMsg"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnSend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>心跳回复信号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>心跳频率(s)：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="send_angel_beats">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>zzz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="use_angel_beats">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>启用心跳      </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="undoRedoEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>监听地址：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineIP">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>监听端口：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinPort">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>1200</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>心跳接收信号：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="recive_angel_beats">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>zzzzzz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
   <property name="toolTip">
    <string>开始监听</string>
   </property>
  </action>
  <action name="actStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
   </property>
   <property name="text">
    <string>停止通讯</string>
   </property>
   <property name="toolTip">
    <string>停止通讯</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本程序</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/212.bmp</normaloff>:/images/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
   <property name="toolTip">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actHostInfo">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/110.JPG</normaloff>:/images/images/110.JPG</iconset>
   </property>
   <property name="text">
    <string>本机地址</string>
   </property>
   <property name="toolTip">
    <string>获取本机地址</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
